benchmarks/benchmark_dataset.py

```diff
@@ -645,18 +660,24 @@ def load_data(self) -> None:
     def sample(self,
                tokenizer: PreTrainedTokenizerBase,
                num_requests: int,
-               output_len: int = DEFAULT_OUTPUT_LEN,
+               output_len: Optional[int] = DEFAULT_OUTPUT_LEN,
```

Comment:
```suggestion
               output_len: Optional[int] = None,
```
since you're dealing with the `None` case in the code already.